nodejs/dist/binding.gyp

```diff
@@ -6,7 +6,12 @@
         'src/uWS.cpp',
         'src/addon.cpp'
       ],
-      'cflags_cc': [ '-std=c++11', '-fexceptions' ]
+      'cflags_cc': [ '-std=c++11', '-fexceptions' ],
```

Comment:
We need to negate the -fno-exceptions also, add `'cflags_cc!': [ '-fno-exceptions' ]`